pyproject.toml

```diff
@@ -5,7 +5,7 @@ dynamic = ["version", "readme"]
 dependencies = [
     "craft-archives>=1.1.3",
     "craft-cli>=2.4.0",
-    "craft-grammar>=1.1.1",
+    "craft-grammar @ git+https://github.com/canonical/craft-grammar.git@CRAFT-2656-more-types",
```

Comment:
Looks like only @sergiusens and @cmatsuoka have release access for craft-grammar